plugins/inputs/synproxy/synproxy.go

```diff
@@ -0,0 +1,101 @@
+// +build linux
+
+package synproxy
+
+import (
+	"bufio"
+	"fmt"
+	"os"
+	"path"
+	"strconv"
+	"strings"
+
+	"github.com/influxdata/telegraf"
+	"github.com/influxdata/telegraf/plugins/inputs"
+)
+
+type Synproxy struct {
+	// Synproxy stats filename (proc filesystem)
+	statFile string
+}
+
+func (k *Synproxy) Description() string {
+	return fmt.Sprintf("Get synproxy statistics from %s", k.statFile)
```

Comment:
Make this a static string with the default location, we wouldn't want the description to change based on the environment variables set.

Reply:
Done